### DISCUSSION TITLE
How to Remove Surrounding padding

### DISCUSSION THREAD

**Discussion by Shiuend** [patrykandpatrick/vico#705] [Questions]
I use vico 2.0.0-alpha.19.
And I want to combine multiple charts seamlessly. How can I completely remove the surrounding gaps?

![截圖 2024-05-16 上午10 11 18](https://github.com/patrykandpatrick/vico/assets/16371119/3894de21-e425-48b1-9a02-9b117eafe395)

Thanks.

**Comment by Gowsky** [comment]
Hello! A bug made `HorizontalAxis` use tick's length for the axis' horizontal insets even with `VerticalAxis.HorizontalLabelPosition.Inside`. It's fixed in Vico [2.0.0-alpha.21](https://github.com/patrykandpatrick/vico/releases/tag/v2.0.0-alpha.21). 

Are you using an invisible label in your `VerticalAxis` regarding the vertical padding? If so, it adds padding at the top and the bottom. Setting it to `null` will resolve the issue. This will change the number of ticks, but you can set it back to the desired value by using [`AxisItemPlacer.Vertical.count`](https://patrykandpatrick.com/vico/wiki/cartesian-charts/axes).
